registration/src/icp.cpp

```diff
@@ -37,4 +37,15 @@
  *
  */
 
-#include <pcl/registration/icp.h>
```

Comment:
I think this removal here is why gicp doesn't compile without change. Instead keep include here?